### PR TITLE
Rename travis-ci/ directory to ci/

### PR DESCRIPTION
**Pull request by danielocfb**
We are no longer using Travis. As such, we should move away from a lot
of CI functionality located in a folder called travis-ci/. This change
renames the travis-ci/ directory to the more generic ci/.
To preserve backwards compatibility until all "consumers" have
transitioned, we add a symbolic link called travis-ci back. It will be
removed in the near term future.

Signed-off-by: Daniel Müller <deso@posteo.net>

### CODE REVIEW

File: travis-ci [hunk 1]
```diff
@@ -0,0 +1 @@
+ci
```
Comment (mykola-lysenko) [inline travis-ci:1]: is this a symlink?

**Comment by mykola-lysenko** [comment]
Bye bye travis ci!

File: travis-ci (hunk 1 above)
Comment (mykola-lysenko) [reply travis-ci:1]: ok, missed your mention of it in the end, thank you!

**Review by mykola-lysenko — APPROVED** [review]

File: travis-ci (hunk 1 above)
Comment (mykola-lysenko) [reply travis-ci:1]: I guess symlink will be required until travis-ci is removed in https://github.com/libbpf/ci/search?q=travis-ci

Comment (danielocfb) [reply travis-ci:1]: > I guess symlink will be required until travis-ci is removed in https://github.com/libbpf/ci/search?q=travis-ci

Yep, that is correct.

**Pull request merged**